vignettes/quickstart.Rmd

```diff
@@ -30,54 +30,33 @@ In particular the core functions are:
 
 ## The Data
 
-In order to demonstrate the package we will use a simulated dataset consisting of 120 
-patients with 3 visits per patient. The patients will be split into two treatment groups
-A and B. Group A will be our control group and group B our active group. The data is
-simulated such that the treatment effect has a linear slope of 4, i.e. the true treatment
-affect across our 3 visits is 0, 4 and 8. 
+In order to demonstrate the package we will use a publicly available example data set from an antidepressant clinical trial of an active drug versus placebo. The relevant endpoint is the Hamilton 17-item rating scale for depression (HAMD17) which was assessed at baseline and weeks 1, 2, 4, and 6. Study drug discontinuation occurred in 24% (20/84) for the active drug and 26% (23/88) for placebo. All data after study drug discontinuation are missing and there is a single additional intermittent missing observation.
```

Comment:
- I don't think the word "example" is needed ?  
- Same comment as above on the 84 and 88 numbers
- Same comment as above regarding the "intermittent missing observation" remark

Reply:
I will keep the "intermittent missing observation" remard but addressed the other two :)